src/components/wallet/NewMnemonicConfirm.tsx

```diff
@@ -5,6 +5,8 @@ import { Card, Col, Row, Button, Form } from 'antd'
 import { Store } from 'antd/lib/form/interface'
 import { v4 as uuidv4 } from 'uuid'
 
+import { shuffleArray } from '../../helpers/shuffleArrayHelper'
```

Comment:
Similar question[ to previous PR](https://github.com/thorchain/asgardex-electron/pull/249): How about to rename function + module names:
- `helpers/shuffleArrayHelper` -> `helpers/array` or `helpers/list`
- `shuffleArray` -> `shuffle`
?